src/main/java/cs361/battleships/models/Submarine.java

```diff
@@ -10,7 +10,17 @@
 	public Submarine(String kind) {
 		this.health = 2;
 		this.size = 5;
-		this.kind = kind;
+		this.submerged=false;
+		if(kind.equals("SUBMARINE")) this.submerged = true;
+		else this.submerged = false;
+		this.kind = kind.toUpperCase();
```

Comment:
toUpperCase is a good addition, so we can avoid any issues with checking if its a submarine.